lib/services/project-service.ts

```diff
@@ -101,7 +101,7 @@ export class ProjectService implements IProjectService {
 	private async extractTemplate(projectDir: string, realTemplatePath: string): Promise<void> {
 		this.$fs.ensureDirectoryExists(projectDir);
 
-		const appDestinationPath = path.join(projectDir, constants.APP_FOLDER_NAME);
+		const appDestinationPath = this.$projectData.getAppDirectoryPath(projectDir);
```

Comment:
Check usages here. What happens if template has existing `nsconfig.json` inside.

Reply:
Templates currently contain only the contents of the app folder. No option to add nsconfig in template for the moment.